code/modules/power/cable.dm

```diff
@@ -678,9 +678,6 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
 			if (showerror)
 				to_chat(user, "<span class='warning'>You can only lay cables on catwalks and plating!</span>")
 			return
-		if(U.intact)						//can't place a cable if it's a plating with a tile on it
-			to_chat(user, "<span class='warning'>You can't lay cable there unless the floor tiles are removed!</span>")
-			return
 		else
 			// cable is pointing at us, we're standing on an open tile
```

Comment:
The else block here is now useless since the if early returns.

Reply:
5a81dc1

Reply:
Yes you are correct.

I added it back.

2b226e1